src/renderer/components/Form.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <form class="aph-form" @submit.prevent="handleOnSubmit" @keyup.enter="handleOnSubmit">
```

Comment:
I don't think you need keyup here. The default behavior on forms when you hit enter is to fire the submit event

Reply:
It has inconsistent behavior. Using the keyup is consistent. Probably something to do with how we handle enter on the Input component.

Reply:
Could you possibly see a scenario when both events fire? I am assuming you are adding keyup to fill in the gaps where 'submit' doesn't fire - I would assume you *could* see both fire at the same time though?